CITATION.cff

```diff
@@ -37,4 +37,4 @@ message: "If you use this software, please cite it using these metadata."
 identifiers:
   - description: The collection of archived snapshots of all versions of FLAME GPU 2
     type: doi
-    value: "10.5281/zenodo.5428984"
\ No newline at end of file
+    value: "10.5281/zenodo.5428984"
```

Comment:
Are you sure the trailing newline wasn't required? (I've no idea, but some things/people are picky)

Reply:
sorry this was unintentional, added by github's editor, I'll put it back
```suggestion
    value: "10.5281/zenodo.5428984"

```